api/bindings/v1alpha1/endpointbinding_types.go

```diff
@@ -33,13 +33,14 @@ import (
 
 // EndpointBindingSpec defines the desired state of EndpointBinding
 type EndpointBindingSpec struct {
-	// Protocol is the Service protocol this Endpoint uses
+	// Scheme is a user-defined field for endpoints that describe how the data packets
+	// are framed by the pod forwarders mTLS connection to the ngrok edge
 	// +kubebuilder:validation:Required
-	// +kubebuilder:default=`TCP`
-	// +kubebuilder:validation:Enum=TCP
-	Protocol string `json:"protocol"`
+	// +kubebuilder:default=`https`
+	// +kubebuilder:validation:Enum=tcp,http,https,tls
```

Comment:
These should be semicolon delimited https://github.com/ngrok/ngrok-operator/blob/8a82ae0caf9ebd4dd234db29ef9fa44bc827e35a/api/ingress/v1alpha1/ippolicy_types.go#L37